lib/actions.js

```diff
@@ -86,23 +86,38 @@ async function main(projectName, options) {
     ).projectName;
   }
 
-  projectName = projectName.toLowerCase().replace(/ |_/g, '-');
+  setProjectName =
```

Comment:
Maybe we can name `setProjectName` as `projectDisplayName`? 
`setProjectName` can be mistaken as a function that sets project name.

```suggestion
  const projectDisplayName =
```